crates/iota/src/iota_commands.rs

```diff
@@ -1241,12 +1241,25 @@ async fn prompt_if_no_config(
         };
 
         if let Some(env) = env {
-            let keystore_path = wallet_conf_path
-                .parent()
-                .unwrap_or(&iota_config_dir()?)
-                .join(IOTA_KEYSTORE_FILENAME);
+            let keystore_path = match wallet_conf_path.parent() {
+                // Wallet config was created in the current directory as a relative path.
+                Some(parent) if parent.as_os_str().is_empty() => {
+                    std::env::current_dir().context("Couldn't find current directory")?
```

Comment:
Can you change it to `iota config directory` in the error message so it's more clear?

Reply:
`current iota config directory`

Reply:
Or don't you want something in between?

Reply:
Would you be fine with `Couldn't find current directory for iota config`?

Reply:
https://github.com/iotaledger/iota/pull/4936/commits/6f52eb4d053f84e01cd52c3b2b9a5033f6902192